doc-content/enterprise-only/springboot/bus-app-keycloak_proc.adoc

```diff
@@ -0,0 +1,61 @@
+[id='bus-app-keycloak_{context}']
+= Configuring the business application with Keycloak
+
+Configuring business applications to use Keycloak as an authentication and authorization provider
+requires few steps. COMMENT: Why would I want to use Keycloak?
```

Comment:
to enable SSO between your services and to have central place to configure and manage your users and roles.